webviews/components/comment.tsx

```diff
@@ -318,100 +319,45 @@ export function AddComment({ pendingCommentText, state, hasWritePermission, isIs
 	</form>;
 }
 
-export function AddCommentSimple({ pendingCommentText, pendingReviewType, isAuthor }: PullRequest) {
+const COMMENT_METHODS = {
+	comment: 'Comment',
+	approve: 'Approve',
+	requestChanges: 'Request Changes'
+}
+
+export const AddCommentSimple = (pr: PullRequest) => {
 	const { updatePR, requestChanges, approve, comment } = useContext(PullRequestContext);
-	const [isBusy, setBusy] = useState(false);
-	const form = useRef<HTMLFormElement>();
 	const textareaRef = useRef<HTMLTextAreaElement>();
-	let selectedValue: ReviewType = pendingReviewType || ReviewType.Comment;
-
-	const onSubmit = async e => {
-		e.preventDefault();
-		try {
-			setBusy(true);
-			const { body }: FormInputSet = form.current;
-			switch (selectedValue) {
-				case ReviewType.RequestChanges:
-					await requestChanges(body.value);
-					break;
-				case ReviewType.Approve:
-					await approve(body.value);
-					break;
-				default:
-					await comment(body.value)
-			}
-			updatePR({ pendingCommentText: '', pendingReviewType: undefined });
-		} finally {
-			setBusy(false);
-		};
-	};
-
-	const onKeyDown = (e: React.KeyboardEvent) => {
-		if ((e.metaKey || e.ctrlKey) && e.key === 'Enter') {
-			onSubmit(e);
+
+	async function submitAction(selected: string): Promise<void> {
+		const { value } = textareaRef.current;
+		switch (selected) {
+			case ReviewType.RequestChanges:
+				await requestChanges(value);
+				break;
+			case ReviewType.Approve:
+				await approve(value);
+				break;
+			default:
+				await comment(value);
 		}
-	};
+		updatePR({ pendingCommentText: '', pendingReviewType: undefined });
+	}
 
 	const onChangeTextarea = (e: React.ChangeEvent<HTMLTextAreaElement>): void => {
 		updatePR({ pendingCommentText: e.target.value });
 	}
 
-	const onChangeRadioSelection = (e: React.ChangeEvent<HTMLInputElement>): void => {
-		selectedValue = e.target.value as ReviewType;
-		updatePR({ pendingReviewType: e.target.value as ReviewType });
-	}
-
-	return <form id='comment-form'
-		ref={form}
-		className='comment-form main-comment-form'
-		onSubmit={onSubmit}>
+	const availableActions = pr.isAuthor
+		? { 'comment': 'Comment '}
```

Comment:
Why doesn't `{ 'comment': 'Comment '}` more closely match `COMMENT_METHODS`? I would expect it to be `{ comment: 'Comment'}` (no quotes and no space).

Reply:
Thanks, fixed!